include/swift/Sema/ConstraintSystem.h

```diff
@@ -3765,10 +3765,20 @@ class ConstraintSystem {
                                        RememberChoice_t rememberChoice,
                                        ConstraintLocatorBuilder locator,
                                        ConstraintFix *compatFix = nullptr);
-  
-  /// Add a materialize constraint for a pack expansion.
+
+  /// Given a tuple with a single unlabeled element that represents a pack
+  /// expansion (either directly via \c PackExpansionType or through a type
+  /// variable constrained to a pack expansion), materialize the pack expansion
+  /// and return its pattern type as a result. The result is a type variable
+  /// because element of the tuple is not required to be resolved at the time of
+  /// the call and operation is delayed until the element is sufficiently
+  /// resolved (see \c simplifyMaterializePackExpansionConstraint)
+  ///
+  /// \param tupleType A tuple with a single unlabeled element that represents a
+  /// pack expansion. 
+  /// \param locator The locator.
```

Comment:
Nit: for completeness you need to specify `\returns` too :)

Reply:
Yes! I forgot😬